lib/std/setutils.nim

```diff
@@ -35,3 +38,28 @@ template toSet*(iter: untyped): untyped =
   for x in iter:
     incl(result, x)
   result
+
+macro enmRange(enm: typed): untyped = result = newNimNode(nnkCurly).add(enm.getType[1][1..^1])
+
+template fullSet*(T: typedesc): untyped =
+  ## Returns a full built-in set of all valid elements
```

Comment:
```suggestion
  ## Returns a full set of all valid elements.
```
The whole module is about the built-in set, so it's kinda redundant to mention that IMO (ditto for the other cases).